client/Pages/EventTypeDetails/View.elm

```diff
@@ -821,7 +821,8 @@ deletePopup model eventType consumersStore subscriptionsStore appsInfoUrl usersI
                                 Nothing ->
                                     none
                             , p [ class "dc-p" ]
-                                [ text "Think twice, notify all consumers and producers."
+                                [ text "Notify all consumers and producers."
+                                , text "Delete all active subscriptions except the one for nakadi-archiver."
```

Comment:
@thomasabraham you might want to move this text to an environment variable as this details is not pertinent to all users of the open source Nakadi UI project. It's also not true for both staging and live environments.